flow360/component/simulation/primitives.py

```diff
@@ -395,6 +395,27 @@ class GhostSurface(_SurfaceEntityBase):
     )
 
 
+# pylint: disable=missing-class-docstring
+@final
+class GhostSphere(GhostSurface):
+    private_attribute_entity_type_name: Literal["GhostSphere"] = pd.Field(
```

Comment:
I think this can just be `type_name`, doesn't need to be private:
```suggestion
    type_name: Literal["GhostSphere"] = pd.Field(
```